tests/trace/test_mm_bf16_swiglu_reference_correctness.py

```
@@ -0,0 +1,35 @@
+    "shape_kwargs",
+    [dict(M=1, gate_up_size=128, K=128), dict(M=3, gate_up_size=1024, K=6144)],
+)
+def test_mm_bf16_swiglu_reference_correctness(shape_kwargs):
+    """Compare the kernel with its strict BF16-boundary reference."""
+    import flashinfer
+    from flashinfer.trace.templates.gemm import mm_bf16_swiglu_trace
+
+    try:
+        inputs = mm_bf16_swiglu_trace.init(**shape_kwargs)
+        api = flashinfer.mm_bf16_swiglu(**inputs)
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)